mail/templates/welcome_course_run_enrollment/body.html

```diff
@@ -0,0 +1,145 @@
+{% extends "email_base.html" %}
+
+{% block content %}
+<!-- 1 Column Text + Button : BEGIN -->
+<tr>
+  <td style="background-color: #ffffff">
+    <table
+      role="presentation"
+      cellspacing="0"
+      cellpadding="0"
+      border="0"
+      width="100%"
+    >
+      <tr>
+        <td
+          style="
+            padding: 20px;
+            font-family: sans-serif;
+            font-size: 15px;
+            line-height: 20px;
+            color: #555555;
+          "
+        >
+          <p style="margin: 0 0 10px">
+            Dear
+            {{ user.name }},
+          </p>
+          <p style="margin: 0 0 10px">Welcome to MIT xPRO!</p>
+          <p style="margin: 0 0 10px">
+            We're thrilled to have you on board for our upcoming course,
+            <strong>{{ enrollment.run }}</strong>. This is a significant step
```

Comment:
Any specific reason, you used run instead of course?
```suggestion
            <strong>{{ enrollment.run.course }}</strong>. This is a significant step
```

Reply:
I used `enrollment.run` here and in all other instances you mentioned because it shows the course name along with the course run that makes it more descriptive. We've used a similar approach in other templates, which might be the reason. However, if it only needs to be the course name, we can change it accordingly.